controller/templates/statefulset.yaml

```diff
@@ -103,32 +103,38 @@ spec:
           # Jupyter server can even accept connections. However, really long running tasks
           # should be put into an init container.
           livenessProbe:
-            httpGet:
-              httpHeaders:
-                - name: host
-                  value: 127.0.0.1:8888
-              path: {{ path }}
-              port: 8888
+            exec:
+              command:
+                - "curl"
```

Comment:
I thought about this too, but this requires curl to be installed inside the jupyter session, which is actually not always the case (for example `jupyter/minimal-notebook` doesn't have it).

Reply:
Ok yeah that explains the failing tests. I will revert the changes to the probes and turn them back off.